Drivers/Src/CANConfig.c

```diff
@@ -35,6 +35,7 @@ const CANLUT_T CANLUT[MAX_CAN_ID] = {
 	[TEMPERATURE] 					= {NOIDX, DOUBLE}, /**     TEMPERATURE                     **/
 	[ODOMETER_AMPHOURS] 			= {NOIDX, DOUBLE}, /**     ODOMETER_AMPHOURS               **/
 	[ARRAY_CONTACTOR_STATE_CHANGE] 	= {NOIDX, BYTE  }, /**     ARRAY_CONTACTOR_STATE_CHANGE    **/
+    [SLIP_SPEED]                    = {NOIDX, DOUBLE},
```

Comment:
Will we eventually need to write this into ReadTritium?

Reply:
ReadTritium reads in every message that the motor sends us, regardless of what it is. The point of the switch case in ReadTritium is that sometimes we may need to do some conditional logic if the message is a certain thing (such as outputting battery SOC to the display). Every message we get is forwarded to SendCarCAN, including Slip Speed.

The CANbus driver uses this LUT when calling CANbus_Read or CANbus_Send, so we had to add an entry for SLIP_SPEED to the lookup table.

Reply:
Got it, thanks!